### PR TITLE
Security Center - Fix JIT documentation

### PR DESCRIPTION
**Pull request by chlahav**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json [hunk 1]
```diff
@@ -2048,14 +2059,15 @@
...
+        "JitNetworkAccessPolicyInitiatePort": {
             "type": "object",
             "properties": {
                 "number": {
                     "$ref": "#/definitions/PortNumber"
                 },
                 "allowedSourceAddressPrefix": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "Source of the allowed traffic. if omitted, the request will be for the source IP address of the initiate reuqest"
```
Comment (annatisch) [inline specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json]: Some grammar and typo correction:
`"Source of the allowed traffic. If omitted, the request will be for the source IP address of the initiate request." `

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json [hunk 2]
```diff
@@ -1949,12 +1959,13 @@
...
                     "type": "string",
-                    "description": "The IP address range that the port was opened for"
+                    "description": "Mutual exclusive with \"allowedSourceAddressPrefixes\" field. should be IP addess or CIDR. e.g. \"192.168.0.3\" or \"192.168.0.0/16\""
                 },
                 "allowedSourceAddressPrefixes": {
                     "type": "array",
-                    "description": "The IP address ranges that the port was opened for",
+                    "description": "Mutual exclusive with \"allowedSourceAddressPrefix\" field.",
                     "items": {
+                        "description": "IP addess or CIDR. e.g. \"192.168.0.3\" or \"192.168.0.0/16\"",
```
Comment (annatisch) [inline specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json]: Some grammar and typo correction:
`"IP addess or CIDR, for example \"192.168.0.3\" or \"192.168.0.0/16\"."`

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json [hunk 3]
```diff
@@ -1882,15 +1884,23 @@
...
-                    "type": "string"
+                    "type": "string",
+                    "description": "Mutual exclusive with \"allowedSourceAddressPrefixes\" field. should be IP addess or CIDR. e.g. \"192.168.0.3\" or \"192.168.0.0/16\""
+                },
+                "allowedSourceAddressPrefixes": {
+                    "type": "array",
+                    "description": "Mutual exclusive with \"allowedSourceAddressPrefix\" field.",
+                    "items": {
+                        "type": "string",
+                        "description": "IP addess or CIDR. e.g. \"192.168.0.3\" or \"192.168.0.0/16\""
```
Comment (annatisch) [inline specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json]: Some grammar and typo correction:
`"IP address or CIDR, for example \"192.168.0.3\" or \"192.168.0.0/16\"."`

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json [hunk 4]
```diff
@@ -1882,15 +1884,23 @@
                     }
                 },
                 "allowedSourceAddressPrefix": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "Mutual exclusive with \"allowedSourceAddressPrefixes\" field. should be IP addess or CIDR. e.g. \"192.168.0.3\" or \"192.168.0.0/16\""
```
Comment (annatisch) [inline specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json]: Some grammar and typo correction:
`"Mutually exclusive with the \"allowedSourceAddressPrefixes\" parameter. Should be an IP address or CIDR, for example \"192.168.0.3\" or \"192.168.0.0/16\"."`

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json [hunk 5]
```diff
@@ -1882,15 +1884,23 @@
                     }
                 },
                 "allowedSourceAddressPrefix": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "Mutual exclusive with \"allowedSourceAddressPrefixes\" field. should be IP addess or CIDR. e.g. \"192.168.0.3\" or \"192.168.0.0/16\""
+                },
+                "allowedSourceAddressPrefixes": {
+                    "type": "array",
+                    "description": "Mutual exclusive with \"allowedSourceAddressPrefix\" field.",
```
Comment (annatisch) [inline specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json]: Some grammar and typo correction:
`"Mutually exclusive with the \"allowedSourceAddressPrefix\" parameter."`

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json [hunk 6]
```diff
@@ -1949,12 +1959,13 @@
                 },
                 "allowedSourceAddressPrefix": {
                     "type": "string",
-                    "description": "The IP address range that the port was opened for"
+                    "description": "Mutual exclusive with \"allowedSourceAddressPrefixes\" field. should be IP addess or CIDR. e.g. \"192.168.0.3\" or \"192.168.0.0/16\""
```
Comment (annatisch) [inline specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json]: Some grammar and typo corrections:
`"Mutually exclusive with the \"allowedSourceAddressPrefixes\" parameter. Should be an IP address or CIDR, for example \"192.168.0.3\" or \"192.168.0.0/16\"." `

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json [hunk 7]
```diff
@@ -1949,12 +1959,13 @@
                 },
                 "allowedSourceAddressPrefix": {
                     "type": "string",
-                    "description": "The IP address range that the port was opened for"
+                    "description": "Mutual exclusive with \"allowedSourceAddressPrefixes\" field. should be IP addess or CIDR. e.g. \"192.168.0.3\" or \"192.168.0.0/16\""
                 },
                 "allowedSourceAddressPrefixes": {
                     "type": "array",
-                    "description": "The IP address ranges that the port was opened for",
+                    "description": "Mutual exclusive with \"allowedSourceAddressPrefix\" field.",
```
Comment (annatisch) [inline specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json]: Some grammar and typo correction:
`"Mutually exclusive with the \"allowedSourceAddressPrefix\" parameter."`

**Review by annatisch — CHANGES_REQUESTED** [review]

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json (hunk 7 above)
Comment (chlahav) [reply specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json]: Resolved

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json (hunk 6 above)
Comment (chlahav) [reply specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json]: Resolved

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json (hunk 5 above)
Comment (chlahav) [reply specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json]: Resolved

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json (hunk 4 above)
Comment (chlahav) [reply specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json]: Resolved

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json (hunk 3 above)
Comment (chlahav) [reply specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json]: Resolved

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json (hunk 2 above)
Comment (chlahav) [reply specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json]: Resolved

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json (hunk 1 above)
Comment (chlahav) [reply specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json]: Resolved

**Review by annatisch — APPROVED** [review]

**Comment by annatisch** [comment]
Thanks @chlahav!
Changes look good to me.
Just to confirm - you're aware that this PR includes breaking changes that will require a major version bump of any published SDKs?

**Comment by chlahav** [comment]
Yes, we don't have any published SDK yet.
In fact, we found these errors while testing an SDK before publishing it 🙂

**Pull request merged**